### ISSUE TITLE
[BUG]: ytfzf: 1821: scrape_yt_search: not found

### ISSUE THREAD

**Issue by anishmittal2020** [pystardust/ytfzf#524]
### To Reproduce

```bash
$ ytfzf --show-thumbnails  Ted
/home/anish/opt/bin/ytfzf: 1821: scrape_yt_search: not found
invalid scraper: yt_search
```

### Information
 - OS: Linux Mint 20.3 (Ubuntu 20.4)
 - Terminal: gnome terminal (bash shell)
 - Ytfzf version: 2.2
- Output of `ls -l "$(which sh)"` (if your using fish: `ls -l (which sh)`):

```
$ ls -l "$(which sh)"
lrwxrwxrwx 1 root 4 Jan  4 15:37 /usr/bin/sh -> dash*
```

**Comment by Euro20179** [comment]
Do you have a config, if so whats in it?

`ytfzf -cY ...` should work

**Comment by anishmittal2020** [comment]
I have installed all dependencies:

```bash
$ type jq
jq is /home/linuxbrew/.linuxbrew/bin/jq

$ type curl
curl is /usr/bin/curl
curl is /bin/curl
curl is /home/linuxbrew/.linuxbrew/bin/curl

$ type mpv
mpv is /home/linuxbrew/.linuxbrew/bin/mpv

$ type fzf
fzf is /home/linuxbrew/.linuxbrew/bin/fzf
fzf is /home/linuxbrew/.linuxbrew/opt/fzf/bin/fzf

$ type yt-dlp 
yt-dlp is /home/linuxbrew/.linuxbrew/bin/yt-dlp

$ type ueberzug
ueberzug is /home/nikhil/.local/bin/ueberzug
```
---

`ytfzf -t` works but shows no thumbnails:

![image](https://user-images.githubusercontent.com/42092626/162634376-02e6dfb8-ecc0-423b-b8b0-dba652500de0.png)

---

Config file

```bash
video_player="vlc"
audio_player="vlc"
```

**Comment by Euro20179** [comment]
ueberzug inst installed on the same isr as everything else, you should try `sudo pip3 install ueberzug`

**Comment by Euro20179** [comment]
ueberzug inst installed on the same isr as everything else, you should try `sudo pip3 install ueberzug`


edit:
just realized your condig peobably doesnt do what you want, you probably want something more like this


```sh

audio_player () {
    vlc "$@"

}

video_player (){
    vlc "$@"
}
```

**Comment by anishmittal2020** [comment]
> ueberzug inst installed on the same place as everything else, you should try `sudo pip3 install ueberzug`

Instead of running the shell script installed in /usr/local/bin, I ran the shell script at the cloned repository as follows:

![image](https://user-images.githubusercontent.com/42092626/162648948-fb09e6d2-c9af-4eee-b3df-3189ca623a30.png)

even this doesn't shows the thumbnail. Thumbnail was displayed with my current dependencies installed with ytfzf version 1.x

**Comment by Euro20179** [comment]
It seems your missing the `python.h` header file.

Try `sudo apt install python3-devel python-devel` and try installing ueberzug again.

**Comment by anishmittal2020** [comment]
I successfully installed ueberzug as you suggested, but I still don't get any thumbnail. May be ytfzf could print some helpful debug message to identify what is going wrong? OR what do you suggest?

![image](https://user-images.githubusercontent.com/42092626/162688250-b69cc7f7-5691-49ce-9f9e-0ceed10f8037.png)

**Comment by mizzunet** [comment]
Same here
![image](https://user-images.githubusercontent.com/10193999/163716982-a61a5b00-5b48-4c89-8a10-32f82b18ae13.png)
 config
```
#ytdl_pref="139+133"
cache_dir="/tmp"

video_pref="133"
audio_pref="139"

fzf_preview_side=right
thumbnail_viewer=chafa 
show_thumbnails=1
thumbnail_quality=start
is_loop=1

ytdl_path=/usr/bin/yt-dlp

**Comment by Euro20179** [comment]
What's the output of `ls -l $(which sh)`, and whats youre ytfzf version?

**Comment by mizzunet** [comment]
@Euro20179 
󰅂 ls -l $(which sh)
lrwxrwxrwx 4 root  9 Jan 02:36  /usr/bin/sh -> bash


2.2.1.rc-6

**Comment by mizzunet** [comment]
I noticed that, ytzf tries to render thumbnail(chafa)

![image](https://user-images.githubusercontent.com/10193999/163814405-6d5d1653-cf3f-4088-a77b-5365acedf0cb.png)

**Comment by dnkl** [comment]
> I just tried on alacritty, thumbail displays. So I guess it's foot's own issue.

Not really. Chafa enables sixel by default on foot, which ytfzf doesn't support(?). Alacritty doesn't support sixel, and there chafa uses plain ASCII "graphics", which _does_ work.

If you can configure the chafa command line, add `--format=symbols` to force ASCII graphics.

**Comment by mizzunet** [comment]
Hmm, it seems chafa works on `foot`
![image](https://user-images.githubusercontent.com/10193999/163820743-03136ff6-c6b7-44d2-a97e-05390fd7c6e9.png)

> If you can configure the chafa command line, add --format=symbols to force ASCII graphics.

I'm not sure if I can pass arguments for `$thumbnail_viewer `

**Comment by Euro20179** [comment]
> which ytfzf doesn't support(?).

fzf doeanr support sixep, therefore ytzf doesnt support it

**Comment by GhoulBoii** [comment]
Hi! Sorry for commenting on a 5 month old issue but [this is a unfixable issue of ueberzug](https://github.com/seebye/ueberzug/wiki/Troubleshooting/119e30f331799b30fb9594db29740685cb09425b#images-appear-on-the-wrong-terminal-window) with certain terminals, one of them being gnome-terminal.

**Comment by chris-aeviator** [comment]
> Try using the development version, that should give an error if ueberzug isn't installed or something
> 
> ```shell
> git clone -b development https://github.com/pystardust/ytfzf
> cd ytfzf
> ./ytfzf -t ...
> ```
> 
> If ueberzug still doesn't work try running this script in a place with .png files
> 
> ```shell
> #!/bin/sh
> 
> files="$(printf "%s\n" *.png)"
> 
> fifo="ueberzug-fifo"
> mkfifo "$fifo"
> ueberzug layer --parser json < "$fifo" &
> exec 3> "$fifo"
> 
> while read -r file; do
>     {
>         printf "{"
>         printf '"%s": "%s",' \
>             'action' 'add' \
>             'identifier' 'f' \
>             'path' "$file" \
>             'x' "0" \
>             'y' "0" \
>             'scaler' 'fit_contain' \
>             'width' "20"
>         printf '"%s": "%s"' 'height' "20"
>         printf "}\n"
>     } > "$fifo"
>     sleep 1
> done <<EOF
> $files
> EOF
> ```
> 
> The images should appear in the top left corner of the terminal
> 
> About the loading of videos, I'm not sure why it's doing that.

when running this script I can see 

    {"type": "error", "name": "AttributeError", "message": "module 'PIL.Image' has no attribute 'ANTIALIAS'"}

. PIL ANTIALIAS has been deprecated and might be a source of issues
